ee/apps/command/server/trpc/trpc.ts

```diff
@@ -0,0 +1,27 @@
+import { TRPCError, initTRPC } from '@trpc/server';
+import superjson from 'superjson';
+import type { Context } from './createContext';
+// import { getRequestIP } from 'h3';
+
+export const trpcContext = initTRPC
+  .context<Context>()
+  .create({ transformer: superjson });
+
+const isAccountAuthenticated = trpcContext.middleware(({ next, ctx }) => {
+  if (!ctx.account || !ctx.account.id || !ctx.account.username) {
+    throw new TRPCError({
+      code: 'UNAUTHORIZED',
+      message: 'You are not logged in'
+    });
+  }
+  return next({
+    ctx: { ...ctx, account: ctx.account }
```

Comment:
why are we doing this?
this is not necessary, as far as I can see

Reply:
additional layer of checks

Reply:
I was talking about the `{ctx:{...ctx}}`, We don't need it

Reply:
this is how we do it in platform and other places
i think its as per trpc docs
but yes, does make more sense